Various technical notes/transition_infos.json

```diff
@@ -557,6 +546,12 @@
         }
       ]
     },
+    {
+      "area_id": "0x72AD42FA",
+      "area_name": "St. Claire's Excavation Camp (Night)",
+      "default_entrance": "0xBA9370DF",
+      "exits": []
+    },
```

Comment:
If you're adding this back in, are you handling the logic that this should be considered the exact same level as day time for entrances ?

Reply:
Technically st claire day & night exit into each other. If that's not something we want handled by rando implementation, it should at least be noted in a comment why their exists are not included.

Reply:
Both exits are put back in the json file, but randomizer is not affected.